dotnet/src/Connectors/Connectors.Google/Extensions/GoogleAIServiceCollectionExtensions.cs

```diff
@@ -47,12 +50,14 @@ public static class GoogleAIServiceCollectionExtensions
     /// <param name="services">The service collection to add the Gemini Embeddings Generation service to.</param>
     /// <param name="modelId">The model for embeddings generation.</param>
     /// <param name="apiKey">The API key for authentication Gemini API.</param>
+    /// <param name="apiVersion">The version of the Google API.</param>
     /// <param name="serviceId">Optional service ID.</param>
     /// <returns>The updated service collection.</returns>
     public static IServiceCollection AddGoogleAIEmbeddingGeneration(
         this IServiceCollection services,
         string modelId,
         string apiKey,
+        GoogleAIVersion apiVersion = GoogleAIVersion.V1_Beta, // todo: change beta to stable when stable version will be available
```

Comment:
@RogerBarreto can we create an issue for this and put in the future sprint so we don't forget to do this?

Reply:
We should be using the `Latest` as the default instead of providing an specific in here, this way this would be used as default instead of a specific enumerable, so every new version there's just one place for changes.

Similar as I pointed out with Azure SDK before in [here](https://github.com/Azure/azure-sdk-for-net/blob/d12489ab27079466308dbd9dcc89879be7f02fe7/sdk/openai/Azure.AI.OpenAI/src/Generated/OpenAIClientOptions.cs#L16C9-L16C89).

Will create an issue to get this updated as well as to keep track of versions

Reply:
In this case we should clearly call out in the XML doc for the parameter that it will always default to the latest, so that consumers know that they need to specify the version if they do not wish to be upgraded silently with a package version update.

Reply:
@westey-m Good point, added to the issue your observation.